### PR TITLE
Remove useless `cli run` parameter `--prompt`

### PR DESCRIPTION
**Pull request by arouene**
`--prompt` arg is not used, if ARGS exists it will be used as a prompt.

### CODE REVIEW

**Comment by ericcurtin** [comment]
@arouene needs a rebase and a re-push to run through CI properly... Would like to see a green build here... For example we may have tests that break because of removal of this, which may need altering

**Comment by rhatdan** [comment]
You also need to remove --prompt definition in docs/ramalama-run.1.md man page.

File: ramalama/cli.py [hunk 1]
```diff
@@ -351,10 +351,9 @@ def _name():
 
 def run_parser(subparsers):
     parser = subparsers.add_parser("run", help="run specified AI Model as a chatbot")
-    parser.add_argument("--prompt", dest="prompt", action="store_true", help="modify chatbot prompt")
     parser.add_argument("-n", "--name", dest="name", help="name of container in which the Model will be run")
     parser.add_argument("MODEL")  # positional argument
-    parser.add_argument("ARGS", nargs="*", help="additional options to pass to the AI Model")
+    parser.add_argument("ARGS", nargs="*", help="Overrides the default prompt")
```
Comment (ericcurtin) [inline ramalama/cli.py:356]: It actually does more than this, it turns off conversation mode, so a response is given and ramalama is exited.

We also may want a --system-prompt like CLI option in future to tell the model how to behave but that's kinda irrelevant to this PR.

@rhatdan introduced this option, so may have opinions on the description.

Comment (rhatdan) [reply ramalama/cli.py:356]: The goal of this option was to have a command executed against the AI Model and then exit.  

Something like the following:

ramalama run MODEL Generate a test framework for python > /tmp/output

As opposed to standard where we end up in a chatbot.

So if I give an argument to the chatbot, process it and exit.  If I don't then go into interactive mode.

Handling this in a different manner would be fine with me as well.

Comment (arouene) [reply ramalama/cli.py]: > We also may want a --system-prompt like CLI option in future to tell the model how to behave but that's kinda irrelevant to this PR.

I saw an issue in llama.cpp that consider removing `--system-prompt`: https://github.com/ggerganov/llama.cpp/issues/9811

Comment (arouene) [reply ramalama/cli.py]: I changed the help message for ARGS, to be a bit more specific

Comment (arouene) [reply ramalama/cli.py]: What could be neat, but in another PR. It's accepting stdin like so : `cat file.py | ramalama run model "check this following code"`

Comment (ericcurtin) [reply ramalama/cli.py]: When I say system prompt, I mean this in llama-cli: `-p "You are a helpful assistant"`. The -p option is effectively a system prompt when used like this.

Comment (ericcurtin) [reply ramalama/cli.py]: I would definately appreciate if this feature was added in a PR @arouene :

> What could be neat, but in another PR. Is accepting stdin like so : cat file.py | ramalama run model "check this following code"

we should also consider this syntax if it makes implementation/usage simpler:

cat file.py | ramalama run model "check this following code" -

Some tools use '-' to denote, read from stdin

**Review by ericcurtin — APPROVED** [review]

File: ramalama/cli.py (hunk 1 above)
Comment (rhatdan) [reply ramalama/cli.py]: I think it is fine to have ramalama act differently if STDIN is not a TTY and read from stdin.

**Pull request merged**